packages/iota/examples/native-token-tour/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "native-token-tour",
+    "version": "1.0.0",
+    "main": "dist/index.js",
+    "private": true,
+    "scripts": {
+        "dist": "tsc",
+        "start": "node dist/index"
+    },
+    "dependencies": {
+        "@iota/crypto.js": "file:../../../../packages/crypto",
+        "@iota/iota.js": "file:../../../../packages/iota",
+        "@iota/util.js": "file:../../../../packages/util",
```

Comment:
Looks like pow-neon is missing: "@iota/pow-neon.js": "file:../../../../packages/pow-neon"?